interface/main/tabs/menu/menus/standard.json

```diff
@@ -904,8 +904,8 @@
         "children": [
```

Comment:
also need to copy this file (to bring over the changes) sites/default/documents/custom_menus/Custom.json